Devall/Tabatadze/view/frontend/web/js/details.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'uiComponent',
+    'ko',
+    'jquery'
+], function (Component, ko, $) {
+    return Component.extend({
+        initialize: function (config) {
+            var feature = this;
+            this._super();
+            this.country(config.countrycompany);
+            this.street(config.streetcompany);
+            this.streetNum(config.numstreet);
+            this.companySize(config.sizecompany);
+            $('select').on('change', function () {
+                var valueSelected = this.value - 1;
```

Comment:
Why the -1? What is it?

Reply:
i'm doing get request on all companies, Array indexes starts with 0 company ids with 1
![image](https://user-images.githubusercontent.com/79899333/116575546-c9397b80-a91f-11eb-9887-50f7582b2014.png)